### DISCUSSION TITLE
Can run node but not npm on Windows 10/11 PowerShell

### DISCUSSION THREAD

**Discussion by andradei** [nodejs/help#4772] [Q&A]
I tried installing `node` and `npm` (shipped with node) with each of these tools (one at a time)
- Scoop
- Winget
- Direct download from official site

And I have the same issue with all of them where:
- Node runs fine on both `Command Prompt` and `PowerShell`
- `npm` works only on `Command Prompt` but **not** on `PowerShell`

![image](https://github.com/nodejs/node/assets/2653546/f97f344e-3613-4270-be56-c2f60848d315)

How do I make `npm` work on PowerShell?

**Comment by bycop** [comment]
Hi, this topic on StackOverflow should help you https://stackoverflow.com/questions/19569990/npm-install-doesnt-work-in-windows-powershell
I think it's how the terminal is handling path/whitespace on path
